### DISCUSSION TITLE
paddleOCR是否支持统信和麒麟的操作系统，如果支持的话部署方式有哪些部署，可以采用C++源码部署吗

### DISCUSSION THREAD

**Discussion by zhaocanglong** [PaddlePaddle/PaddleOCR#12465] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

**Comment by gl94** [comment]
您好 请问成功了吗?
